src/cli.js

```diff
@@ -1,56 +1,65 @@
 #!/usr/bin/env node
 // @flow
 import meow from 'meow'
-import updateNotifier from 'update-notifier'
 
 import pkg from '../package.json'
 import commands from './commands'
 import FLAGS from '@constants/flags'
 import findGitmojiCommand from '@utils/findGitmojiCommand'
 
-updateNotifier({ pkg }).notify({ isGlobal: true })
-
-const cli = meow(
-  `
-  Usage
-    $ gitmoji
-  Options
-    --${FLAGS.COMMIT}, -c    Interactively commit using the prompts
-    --${FLAGS.CONFIG}, -g    Setup gitmoji-cli preferences.
-    --${FLAGS.INIT}, -i      Initialize gitmoji as a commit hook
-    --${FLAGS.LIST}, -l      List all the available gitmojis
-    --${FLAGS.REMOVE}, -r    Remove a previously initialized commit hook
-    --${FLAGS.SEARCH}, -s    Search gitmojis
-    --${FLAGS.UPDATE}, -u    Sync emoji list with the repo
-    --${FLAGS.VERSION}, -v   Print gitmoji-cli installed version
-  Examples
-    $ gitmoji -l
-    $ gitmoji bug linter -s
-`,
-  {
-    flags: {
-      [FLAGS.COMMIT]: { type: 'boolean', alias: 'c' },
-      [FLAGS.CONFIG]: { type: 'boolean', alias: 'g' },
-      [FLAGS.HELP]: { type: 'boolean', alias: 'h' },
-      [FLAGS.INIT]: { type: 'boolean', alias: 'i' },
-      [FLAGS.LIST]: { type: 'boolean', alias: 'l' },
-      [FLAGS.REMOVE]: { type: 'boolean', alias: 'r' },
-      [FLAGS.SEARCH]: { type: 'boolean', alias: 's' },
-      [FLAGS.UPDATE]: { type: 'boolean', alias: 'u' },
-      [FLAGS.VERSION]: { type: 'boolean', alias: 'v' }
+type Options = { [$Values<typeof FLAGS>]: Function }
+
+const main = async () => {
+  const { default: updateNotifier } = await import('update-notifier')
```

Comment:
I'm not a big fan of this workaround to import ESM in CJS 😅

I have a branch in the repo migrating everything to ESM and updating the dependencies but Jest is blocking it as the support for this is still not there.

Reply:
> I have a branch in the repo migrating everything to ESM and updating the dependencies but Jest is blocking it as the support for this is still not there.

I tried to migrate the repo to ESM and ran into a problem with Jest too. I almost got everything to work. However, I wasn't able to load the `package.json` file in Jest.

Nothing seemed to work. I tried using import assertions but Jest wasn't able to parse it. I tried using `fs.readFileSync` but wasn't able to get that to work either. Relative imports require `import.meta.url` but Jest didn't allow that.

So, in the end I decided to use dynamic imports as a workaround for importing ESM into the CJS package.

Reply:
> I will try to see if I can unblock the branch I have to fully migrate the cli into ESM without having to do workarounds
> 
> If I'm not able to solve the tests issues I'll come back to this PR 😊

Sounds excellent. Let me know whether I can help in any way.

Reply:
Hey! I managed to migrate everything to ESModules (including update notifier) however I'm having some issues with specific jest unit tests, you can take a look at the branch here:

https://github.com/carloscuesta/gitmoji-cli/tree/esmodules-migration

I'll try to see if I can solve these issues, another option would be to migrate to Vite or something like that 😊

Reply:
Sure, I'll look into the Jest issue today.

Reply:
Hey @carloscuesta, I fixed the tests in your `esmodules-migration` branch and created a new PR, #925, for it. If it is satisfactory then we can merge that PR and close this one.